src/mission_companion.cpp

```diff
@@ -185,28 +185,28 @@ bool talk_function::outpost_missions(npc *p, std::string id, std::string title)
     std::string entry, entry_aux;
 
     if (id == "SCAVENGER"){
-        col_missions["Assign Scavenging Patrol"] = "Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
+        col_missions["Assign Scavenging Patrol"] = _("Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
```

Comment:
I don't follow your first question, do you mean should the strings be
restructured to make translation easier?
You can assign missions to NPCs with the debug menu, but I think it's not
reliable, they act strange in various ways when you do it.  When I test
missions, I generally just use debug features to make things easy, like
making myself invincible, revealing the map, and teleporting instead of
walking.